src/main/webapp/journeys.html

```diff
@@ -12,16 +12,19 @@
     <script src="common.js"></script>
     <script src="app.js"></script>
     <link href="style.css" rel="stylesheet" type="text/css" media="screen" />
+    <script src="mapsScript.js"></script>
   </head>
   
-  <body onload="displayJourneys()">
-
+  <body onload="loadMapsScript();displayJourneys()">
```

Comment:
You might want to consider using `DOMContentLoaded` - `onload` waits for *everything* including images and stylesheets etc, but `DOMContentLoaded` waits only for the DOM and scripts.

```html
<script>
  document.addEventListener('DOMContentLoaded', () => {
    loadMapsScript();
    displayJourneys();
  });
</script>


```

Reply:
Right, done.